src/spasht/ui/template/agent_spasht.js.twig

```diff
@@ -8,17 +8,21 @@
 
 <script>
 $("#advanceSearchForm").ready(function(){
-    // Hide Advance Search Form
-    $("#advanceSearchForm").hide();
-});
+  $("#advanceSearchForm").hide();
+  $(".ui-icon-caret-1-n").hide();
+})
 
-function advanceSearchEnabled() {
-  if (document.getElementById('advanceSearch').checked == true) {
-    $("#advanceSearchForm").show(700);
-    $("#definitionsTable").hide(400);
+function advanceSearchFormEnabled(){
+  if (document.getElementById("advanceSearchFormStatus").value === "hidden") {
+    $(".ui-icon-caret-1-s").hide();
+    $(".ui-icon-caret-1-n").show();
+    $("#advanceSearchForm").show(300);
```

Comment:
Adding a `blind` effect to the `show()` and `hide()` of `#advanceSearchForm` will make it look more natural. What do you think?

Reply:
Changed in new Commits